apps/src/gamelab/blocks.js

```diff
@@ -412,21 +412,30 @@ export default {
         this.appendDummyInput()
             .appendTitle(Blockly.Msg.VARIABLES_GET_TITLE)
             .appendTitle(Blockly.disableVariableEditing ? fieldLabel
-                : new Blockly.FieldVariable(Blockly.Msg.VARIABLES_GET_ITEM), 'VAR')
+                : new Blockly.FieldVariable(
+                    Blockly.Msg.VARIABLES_SET_ITEM,
+                    null,
+                    null,
+                    Blockly.BlockValueType.SPRITE,
+                  ),
```

Comment:
Should FieldVariable be updated to just take an options object?

Reply:
Oh that's a good idea, added that to my list.